### ISSUE TITLE
add a "modules_dir" setting in digger.yml to make include patterns related to modules dynamic

### ISSUE THREAD

**Issue by fleroux514** [diggerhq/digger#393]
We are using a mono-repo, where different projects are reusing various modules specified under the `./modules` directory which contains 10+ modules.

In each project, I currently need to do a manual verification of which modules are being used, in order to specify them statically using the `include_patterns` project config. And if we eventually start using a new Terraform module in a project, there is a good change we might forget to update digger.yml accordingly while doing so. So the current config in very error prone.

Digger should make this dynamic by specifying a `modules_dir` project setting:
```
projects:
  - name: dev
    dir: ./envs/dev
    modules_dir: ./modules
```

It could scan the project code and detect that the "vpn-ha" module is being used in this case, instead of having to manually include it using `include_patterns`:

./envs/dev/main.tf
```
module "shared_base_vpn" {
  source = "../../modules/vpn-ha"

  project_id           = local.base_project_id
  default_region1      = var.default_region1
  default_region2      = var.default_region2
...
}
```

**Comment by ZIJ** [comment]
Thanks for reporting it @fleroux514 ! 
Indeed, modules_dir making sense
Tagging @Spartakovic if you have any input on the way this should work?

**Comment by ZIJ** [comment]
The last comment suggests the issue can be closed. Labeling as such - if someone disagrees please indicate in the next few days

**Comment by fleroux514** [comment]
Correct, just updated the title/description for this feature request.
